README.md

```diff
@@ -290,6 +290,11 @@ With:
     <VERVION> = Id used to identify the progressive release
     <PERCENTAGE> = % of devices which will get the new release. The % should be increase by 25 every 6 hours when the snap store team validates the release is going well. The values for this should be: 25, 50, 75 and 100.   
 
+After many hours, once the progressive release has reached 100% please do:
```

Comment:
s/once the progressive release has reached/ONCE THE PROGRESSIVE RELEASE HAS REACHED/